### PR TITLE
Remove NuGet download

### PR DESCRIPTION
**Pull request by fredericDelaporte**
NAnt is not set to download over TLS 1.2, causing the Nuget download to fail.

### CODE REVIEW

File: Tools/packages.csproj [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <ItemGroup>
+    <Compile Remove="**\*" />
+    <EmbeddedResource Remove="**\*" />
+    <None Remove="**\*" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CSharpAsyncGenerator.CommandLine" Version="0.17.1" />
+    <PackageReference Include="NUnit.Console" Version="3.10.0" />
+    <PackageReference Include="NUnit.ConsoleRunner" Version="3.10.0" />
```
Comment (fredericDelaporte) [inline Tools/packages.csproj:16]: Required for `get-tool-info` task.

File: buildcommon.xml [hunk 2]
```diff
@@ -1,179 +1,170 @@
-<?xml version="1.0" ?>
```
Comment (hazzik) [inline buildcommon.xml:1]: Why the whole file has changed?

Comment (fredericDelaporte) [reply buildcommon.xml:1]: No idea.
The spacing is the same, and it is an UTF-8 CRLF file on master and on that branch.
But GitHub see some whitespace change on each line nonetheless.

Comment (fredericDelaporte) [reply buildcommon.xml:1]: It seems this had already happened previously as suggested by [this comment](https://github.com/nhibernate/NHibernate-Caches/pull/74#issue-308386205), but a quick look in the history has not shown me when.

**Pull request merged**